sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/events/AcsEventType.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models.events;
+
+import com.azure.communication.callingserver.implementation.models.AcsEventTypeInternal;
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Collection;
+
+/** Defines values for AcsEventType. */
+public final class AcsEventType extends ExpandableStringEnum<AcsEventType> {
```

Comment:
AcsEventType is now hidden from .NET, per discussion. I am wondering if we can do similar thing with Java?

Reply:
Yes, I haven't updated it yet as yesterday was working on something else. I will sync the change.

Reply:
fixed